tests/test_modeling_tf_transfo_xl.py

```diff
@@ -364,7 +364,7 @@ def test_lm_generate_transfo_xl_wt103(self):
                     0,
                 ]
             ],
-            dtype=tf.int31,
+            dtype=tf.int32,
```

Comment:
😂

Reply:
Thought you can save memory like this :D